### PR TITLE
Resolves dependency issues for issue #30

### PR DESCRIPTION
**Pull request by troy0820**
+ Created a .bithoundrc to ignore/mute dependencies for files that we are continuously updated, i.e **trailpack, eslint-config-trails,etc** 
+ This also gets rid of the word count error that comes up in the index.js at the root of trails.

### CODE REVIEW

File: .bithoundrc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "critics": {
+   "wc": { "limit": 5000 }
+    },
```
Comment (tjwebb) [inline .bithoundrc]: Indentation fail? Can't tell if github UI problem

**Comment by tjwebb** [comment]
Thanks, now I know how bithoundrc works :)

**Comment by tjwebb** [comment]
@troy0820 can you take another look at this? I think getting bithound working will clean up a lot of the PRs outstanding.

This is still failing: 
>  bitHound - Code — 37 failing files.

**Comment by troy0820** [comment]
Yeah, I'll take a look at this. @tjwebb

**Comment by tjwebb** [comment]
@troy0820 awesome, thanks.

**Pull request closed without merging**